### ISSUE TITLE
Trailing slash on repo manager

### ISSUE THREAD

**Issue by david4096** [ga4gh/server#1032]
When adding a variant set I got odd behavior if I left the trailing slash.

```
(server-1)➜  server-1 git:(temp-pysam) ✗ python repo_dev.py add-variantset ga4gh-example-data brca1 tests/data/datasets/dataset1/variants/1KG_GRCh37_VEP_edit/                     
Exception for repo at 'ga4gh-example-data'
Path 'ga4gh-example-data/datasets/brca1/variants/' already exists
```

**Comment by dcolligan** [comment]
Good catch.  This has to do with lame basename functionality:
```
>>> import os
>>> os.path.basename('bob')
'bob'
>>> os.path.basename('bob/')
''
```

**Comment by dcolligan** [comment]
Fixed in #1033
